src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -121,6 +121,8 @@ public function load($data, $type = null): RouteCollection
                         '_controller' => $controller,
                         '_format' => null,
                         '_api_resource_class' => $operation['resource_class'],
+                        '_api_input_resource_class' => $operation['input_class'],
```

Comment:
Just a naming thing, but I would call them `api_input_class` and `api_output_class` because they aren't resources anymore, but more "resources representations", or "dtos". I'm not really sure about this, thought.